self_hosting_machinery/scripts/first_run.py

```diff
@@ -20,16 +20,8 @@ def convert_old_configs(model_assigner: ModelAssigner):
     if os.path.exists(openai_watchdog_cfg_fn):
         os.unlink(openai_watchdog_cfg_fn)
 
-    for gpu in range(16):
-        fn = os.path.join(env.DIR_WATCHDOG_D, "model-gpu%d.cfg" % gpu)
-        if not os.path.exists(fn):
-            continue
-        text = open(fn).read()
-
-    model_assigner.models_to_watchdog_configs()
```

Comment:
this is needed if we remove something from models list (deprecated or something)